Server/AIServer/ServerDlg.cpp

```diff
@@ -1177,8 +1196,11 @@ BOOL CServerDlg::MapFileLoad()
 			{
 				if (!pMap->LoadRoomEvent(row.RoomEvent))
 				{
-					errormsg.Format(_T("Map Room Event Load 실패 - %s\n"), szFullPath.GetString());
-					AfxMessageBox(errormsg);
+					std::string error = std::format("MapFileLoad: LoadRoomEvent failed: {}", mapPath.generic_string());
```

Comment:
Okay, because this one's a path it gets messy. But here's what you can do.
Make the base error a wide string. Fetch the path as a native_string() (which is std::wstring) and convert the entire error with WideToUtf8() from shared/StringConversion.h

Then pass the utf-8 error to spdlog, and pass the wide error to AfxMessageBox.